lib/utils.js

```diff
@@ -133,3 +131,31 @@ normalizeBinary.appNames = {
 }
 
 exports.normalizeBinary = normalizeBinary;
+
+function findMacAppByChannel(channel, opt) {
+  // Try to find an installed app on Mac OS X for this channel.
+  opt = opt || {
+    spawnSync: defaultSpawnSync,
+  };
+  // Example: mdfind "kMDItemCFBundleIdentifier == 'org.mozilla.nightly'"
+  var results = opt.spawnSync("mdfind", ["kMDItemCFBundleIdentifier == 'org.mozilla." + channel + "'"]);
+  var result = false;
```

Comment:
If we make this `null`, then we can just assign it at the caller, as per the other comment.